File: PRAK305-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

void konversi_detik(int total_detik) {
    int detik, menit, jam, hari;

    detik = total_detik % 60;
    menit = (total_detik / 60) % 60;
    jam = (total_detik / 3600) % 24;
    hari = total_detik / 86400;

    if (hari > 0) {
        printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, detik);
    } else {
        printf("%02d:%02d:%02d\n", jam, menit, detik);
    }
}

int main() {
    int detik, jumlah_input = 5;

    for (int i = 0; i < jumlah_input; i++) {
        scanf("%d", &detik);
        konversi_detik(detik);
    }

    return 0;
}

File: MODUL 3/PRAK305-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
def konversi_detik(total_detik):
    detik = total_detik % 60
    menit = (total_detik // 60) % 60
    jam = (total_detik // 3600) % 24
    hari = total_detik // 86400

    if hari > 0:
        print(f"{hari} hari {jam:02}:{menit:02}:{detik:02}")
    else:
        print(f"{jam:02}:{menit:02}:{detik:02}")

jumlah_input = 5

for _ in range(jumlah_input):
    detik = int(input())
    konversi_detik(detik)
